### ISSUE TITLE
Character number limit for a "new routine" name

### ISSUE THREAD

**Issue by Timbobeek** [vkmicro/GB_IssueTracking#7]
**Describe the suggestion / bug**
Character limit for a "new routine" is set to 16 characters

**Steps to reproduce the behavior**
1. Hit on "New Routine"
2. Tap "New Routine" tab at the top
3. Enter a name

**If Suggestion, write explanation**
I think it would be better to set a very high limit or just make it limitless if possible. I personally hate when there are character limits in apps, but I am no sure if it is difficult mechanically to set up. I have noticed that you have changed it, it used to be 9. 16 is much better than 16 characters.

**Device information**
 - Device: Samsung Galaxy S6 Edge
 - OS: Android 7.0

**Comment by vkmicro** [comment]
Yeah so the problem with that is the difference in phone resolutions :/ some people have low res phone screens and due to the UI size and all the name size limits had to be implemented. 

I might look into making it so that it auto zooms in / out based on resolution of the device. This might help it, but i'm unsure, i'll have to test around with that.
